plugins/doc_fragments/foreman.py

```diff
@@ -105,3 +105,13 @@ class ModuleDocFragment(object):
       - Xenserver
     type: str
 '''
+
+    TAXONOMY = '''
+options:
+  organizations:
+    description: List of organizations the entity should be assigned to
```

Comment:
I did not override the description in the various modules, as I find "the entity" exact enough (plus it would make the doc fragment almost useless, as it only contains description and type.)